profiles/ThiagoTatico.json

```
@@ -0,0 +1,6 @@
+{
+  "image": "https://user-images.githubusercontent.com/86478479/161040763-da61292e-d7b4-4681-bbad-67531331d34f.png",
+  "issueId": 1085,
+  "name": "ThiagoTatico",
```

```suggestion
  "name": "Thiago Tatico",
```
Give a space between your first and last name.